nat/service_ics_windows.go

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package nat
+
+import (
+	"golang.org/x/sys/windows/registry"
+)
+
+func setICSAddresses(ip string) error {
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `System\CurrentControlSet\Services\SharedAccess\Parameters`, registry.QUERY_VALUE|registry.SET_VALUE)
+	if err != nil {
+		return err
+	}
+	defer k.Close()
+
+	if err := k.SetStringValue("ScopeAddress", ip); err != nil {
+		return err
+	}
+	return k.SetStringValue("ScopeAddressBackup", ip)
```

Comment:
I would also set `StandaloneDhcpAddress` to the same address, not sure how defaults would behave if someone would try try get IP through DHCP requets without it.

Reply:
Hm, I don't have this parameter in my test Windows. I'll take a look.

Reply:
Looks like it is missing on Windows 10.

Reply:
In our case DHCP with TUN devices, Wireguard allowed IPs is not the best choice.

Reply:
Changed it to set and restore keys if it originally exists.